### PR TITLE
Add Zed60 PCB

### PR DESCRIPTION
**Pull request by mechlovin**
## Description

Add Zed60 PCB, APM32F103.
Thanks.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechlovin/zed60/chconf.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* Copyright 2022 QMK
```
Comment (zvecr) [inline keyboards/mechlovin/zed60/chconf.h:1]: This file shouldnt be needed any more, as the defaults were updated last breaking changes.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/mechlovin/zed60/chconf.h (hunk 1 above)
Comment (mechlovin) [reply keyboards/mechlovin/zed60/chconf.h]: Deleted, thank you.

File: keyboards/mechlovin/zed60/rules.mk [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/mechlovin/zed60/rules.mk:19]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/mechlovin/zed60/config.h [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+//#define LED_NUM_LOCK_PIN B0
+#define LED_CAPS_LOCK_PIN A4
+//#define LED_SCROLL_LOCK_PIN B2
+//#define LED_COMPOSE_PIN B3
+//#define LED_KANA_PIN B4
+
+//#define BACKLIGHT_PIN B7
+//#define BACKLIGHT_LEVELS 3
+//#define BACKLIGHT_BREATHING
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/config.h:58]:
```suggestion
#define LED_CAPS_LOCK_PIN A4
```

File: keyboards/mechlovin/zed60/config.h [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* define if matrix has ghost (lacks anti-ghosting diodes) */
+//#define MATRIX_HAS_GHOST
+
+/* Bootmagic Lite key configuration */
+//#define BOOTMAGIC_LITE_ROW 0
+//#define BOOTMAGIC_LITE_COLUMN 0
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/config.h:101]:
```suggestion
```

File: keyboards/mechlovin/zed60/info.json [hunk 5]
```diff
@@ -0,0 +1,78 @@
+{
+    "keyboard_name": "Zed60",
+    "url": "",
+    "maintainer": "Mechlovin' Studio",
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/info.json:6]:
```suggestion
    "layout_aliases": {
        "LAYOUT_all": "LAYOUT_60_ansi_split_bs_rshift"
    },
    "layouts": {
        "LAYOUT_60_ansi_split_bs_rshift": {
```

File: keyboards/mechlovin/zed60/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+## Bootloader
+
+Enter the bootloader in 4 ways:
+
+* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
+* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
+* **Keycode in layout**: Press the key mapped to `RESET` if it is available
+* **Bootloader**: Hold down the key at (0,13) in the matrix (Backspace) and plug in the keyboard
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/readme.md:27]:
```suggestion
## Bootloader

Enter the bootloader in 4 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
* **Bootloader**: Hold down the key at (0,13) in the matrix (Backspace) and plug in the keyboard 

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/mechlovin/zed60/zed60.h [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+                │30      │32 │33 │34 │35 │36 │37 │38 │39 │3A │3B │3C    │3D │   │3C        │ 2.75u RShift
+ *              ├────┬───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬┴───┤   └──────────┘
+ *              │40  │41  │42  │46                      │4A  │4B  │4C  │4D  │ Standard
+ *              └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+ *              ┌─────┬───┬─────┬───────────────────────────┬─────┬───┬─────┐
+ *              │40   │41 │42   │46                         │4B   │4C │4D   │ Tsangan/WKL
+ *              └─────┴───┴─────┴───────────────────────────┴─────┴───┴─────┘
+ */
+
+#define LAYOUT_all( \
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/zed60.h:39]:
```suggestion
#define LAYOUT_60_ansi_split_bs_rshift( \
```

File: keyboards/mechlovin/zed60/info.json [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+                {"x": 0, "y": 4, "w": 1.25},
+                {"x": 1.25, "y": 4, "w": 1.25},
+                {"x": 2.5, "y": 4, "w": 1.25},
+                {"x": 3.75, "y": 4, "w": 6.25},
+                {"x": 10, "y": 4, "w": 1.25},
+                {"x": 11.25, "y": 4, "w": 1.25},
+                {"x": 12.5, "y": 4, "w": 1.25},
+                {"x": 13.75, "y": 4, "w": 1.25}
+            ]
+        }
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/info.json:76]:
```suggestion
        },
        "LAYOUT_60_ansi": {
            "layout": [
                {"x": 0, "y": 0},
                {"x": 1, "y": 0},
                {"x": 2, "y": 0},
                {"x": 3, "y": 0},
                {"x": 4, "y": 0},
                {"x": 5, "y": 0},
                {"x": 6, "y": 0},
                {"x": 7, "y": 0},
                {"x": 8, "y": 0},
                {"x": 9, "y": 0},
                {"x": 10, "y": 0},
                {"x": 11, "y": 0},
                {"x": 12, "y": 0},
                {"x": 13, "y": 0, "w": 2},

                {"x": 0, "y": 1, "w": 1.5},
                {"x": 1.5, "y": 1},
                {"x": 2.5, "y": 1},
                {"x": 3.5, "y": 1},
                {"x": 4.5, "y": 1},
                {"x": 5.5, "y": 1},
                {"x": 6.5, "y": 1},
                {"x": 7.5, "y": 1},
                {"x": 8.5, "y": 1},
                {"x": 9.5, "y": 1},
                {"x": 10.5, "y": 1},
                {"x": 11.5, "y": 1},
                {"x": 12.5, "y": 1},
                {"x": 13.5, "y": 1, "w": 1.5},

                {"x": 0, "y": 2, "w": 1.75},
                {"x": 1.75, "y": 2},
                {"x": 2.75, "y": 2},
                {"x": 3.75, "y": 2},
                {"x": 4.75, "y": 2},
                {"x": 5.75, "y": 2},
                {"x": 6.75, "y": 2},
                {"x": 7.75, "y": 2},
                {"x": 8.75, "y": 2},
                {"x": 9.75, "y": 2},
                {"x": 10.75, "y": 2},
                {"x": 11.75, "y": 2},
                {"x": 12.75, "y": 2, "w": 2.25},

                {"x": 0, "y": 3, "w": 2.25},
                {"x": 2.25, "y": 3},
                {"x": 3.25, "y": 3},
                {"x": 4.25, "y": 3},
                {"x": 5.25, "y": 3},
                {"x": 6.25, "y": 3},
                {"x": 7.25, "y": 3},
                {"x": 8.25, "y": 3},
                {"x": 9.25, "y": 3},
                {"x": 10.25, "y": 3},
                {"x": 11.25, "y": 3},
                {"x": 12.25, "y": 3, "w": 2.75},

                {"x": 0, "y": 4, "w": 1.25},
                {"x": 1.25, "y": 4, "w": 1.25},
                {"x": 2.5, "y": 4, "w": 1.25},
                {"x": 3.75, "y": 4, "w": 6.25},
                {"x": 10, "y": 4, "w": 1.25},
                {"x": 11.25, "y": 4, "w": 1.25},
                {"x": 12.5, "y": 4, "w": 1.25},
                {"x": 13.75, "y": 4, "w": 1.25}
            ]
        }
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/mechlovin/zed60/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define LT1_CAP     LT(1, KC_CAPS)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/keymaps/default/keymap.c:21]:
```suggestion
    [0] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/mechlovin/zed60/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        LT1_CAP, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, TO(1),
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_MENU, KC_RCTL
+    ),
+    [1] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/keymaps/default/keymap.c:28]:
```suggestion
    [1] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/mechlovin/zed60/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define LT1_CAP     LT(1, KC_CAPS)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/keymaps/via/keymap.c:21]:
```suggestion
    [0] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/mechlovin/zed60/keymaps/via/keymap.c [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        LT1_CAP, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, TO(1),
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_MENU, KC_RCTL
+    ),
+    [1] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/keymaps/via/keymap.c:28]:
```suggestion
    [1] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/mechlovin/zed60/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_P7,   KC_P8,   KC_P9,   _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_P4,   KC_P5,   KC_P6,   KC_PENT,
+        _______,          _______, _______, _______, _______, _______, _______, _______, KC_P0,   KC_P1,   KC_P2,   KC_P3,   _______,
+        _______, _______, _______,                            _______,                   _______, _______, _______, _______
+    ),
+    [2] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/keymaps/via/keymap.c:35]:
```suggestion
    [2] = LAYOUT_60_ansi_split_bs_rshift(
```

File: keyboards/mechlovin/zed60/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,   _______,
+        _______, _______, _______,                            _______,                   _______, _______, _______, _______
+    ),
+    [3] = LAYOUT_all(
```
Comment (noroadsleft) [inline keyboards/mechlovin/zed60/keymaps/via/keymap.c:42]:
```suggestion
    [3] = LAYOUT_60_ansi_split_bs_rshift(
```

**Review by noroadsleft — COMMENTED** [review]
This set of suggestions is optional; the board will still compile if you choose not to include them.

**Comment by noroadsleft** [comment]
There's a [whole other review](https://github.com/qmk/qmk_firmware/pull/17304#pullrequestreview-1039348200) with comments that need addressing.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**